### ISSUE TITLE
Working with a specif number of agents in an evacuation scenario

### ISSUE THREAD

**Issue by OrranaLhaynher** [matsim-org/matsim-code-examples#118]
I'm working in an evacuation scenario from the Woolsey Fire and I'm trying for weeks to generate my population which have a specific number of agents and which has no census data. I tried making my own population script based on RunPPopulationGenerator and RunDemandGenerationFromShapefile and I have found no sucess. I've tried also setting a random start point from my evacuees to the localization of my shelters but it didn't work at all too. I wanted to know how to generate a population file with a specific number of agents without a census data, taking into account a large number of agents.

**Comment by kainagel** [comment]
Could you please provide a bit more context?  Are you working with https://github.com/matsim-org/evacuation-web-app (advantage: more user friendly; disadvantages: working with older version of MATSIM, not supported), or with https://github.com/matsim-org/matsim-code-examples ?  If the latter, then ideally, you would have your project in a public clone/fork of https://github.com/matsim-org/matsim-example-project and give us access to that.

**Comment by OrranaLhaynher** [comment]
I'm working with https://github.com/matsim-org/matsim-code-examples, and now I finally managed to generate my population, but I found another problem. I have a specific amount of shelters, which will be used as the destination for my agents, and I've seen that there's how to create random source and destination points, but how do I do that by creating specific destination points? I had seen that if I had the coordinates I could generate these points easily, however my road network is very large and I do not know yet how to find specific coordinates in it. The code I used to create my population was https://github.com/OrranaLhaynher/matsim-example-project/blob/master/src/main/java/org/matsim/project/RunPopulation.java, with some modifications to fit my problem. Thank you for replying.
